### ISSUE TITLE
[Toolbar] @axe-core/react flags role="toolbaritem" as invalid

### ISSUE THREAD

**Issue by tarngerine** [radix-ui/primitives#949]
### Current Behavior

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/1629901/139925938-5e521cd8-aca8-4887-9359-8203527b8f07.png">

### Expected behavior

should not flag any warnings

### Reproducible example

https://codesandbox.io/s/bold-blackwell-6lmwx?file=/src/App.tsx

### Suggested solution

should prob be radio for ToggleGroup = single and checkbox for ToggleGroup = multi?

### Additional context

not sure if all the toolbar items are valid, might be worth testing them all

### Your environment
see CSB

**Comment by jjenzz** [comment]
> should prob be radio for ToggleGroup = single and checkbox for ToggleGroup = multi?

According to the `toolbar` WAI ARIA example, our `<ToggleGroup multiple>` is correct but it looks like the single one should be using `role=radio`: https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html

I've created a PR to remove the invalid `toolbaritem` roles and I'll create a separate ticket to address the `ToggleGroup` issues when `type=single`.
